zulip_bots/zulip_bots/bots/trivia_quiz/trivia_quiz.py

```diff
@@ -43,7 +43,7 @@ def handle_message(self, message: Dict[str, Any], bot_handler: Any) -> None:
                 bot_handler.send_reply(message, bot_response)
                 return
             quiz = json.loads(quiz_payload)
-            correct, bot_response = grade_question(quiz, answer)
+            correct, bot_response = handle_answer(quiz, answer, quiz_id, bot_handler)
```

Comment:
We should rename `correct` to `start_new_question` here, since the semantics of the variable have changed to that.